acceptance_tests/features/change_response_status.feature

```diff
@@ -3,15 +3,27 @@ Feature: Change response status
   I need to change the case response status to completed by phone for an RU for a specific CE for a specific survey
   So that the respondent can complete the survey over the phone and they do not receive reminders
 
+  Background: Internal user is already signed in
+    Given the internal user is already signed in
```

Comment:
Do we need this given as well as the background?

Reply:
The background line is just text, doesn't trigger any code
I'll make it a bit more verbose though since there's more happening in the background than it says